### PR TITLE
Update Docker files, removed Jenkins files, added Docker to README

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by jpdahlke — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -896,6 +901,81 @@ following articles:
...
+docker exec -it emissary ./emissary env
+```
+
+If you want to connect to the web front-end for Emissary, some additional steps are need to configure jetty. We need to
+give the container a hostname and pass that onto Emissary from the command line. Sample command:
+```
+docker run -it --rm --name emissary --hostname emissary-001 -p 8001:8001 emissary server -a 2 -p 8001 -s http -h emissary-001
+```
+
+Then from a browser, go to http://localhost:8001/ to see the endpoints.
```
Comment (LightSeekerSC) [inline README.md:964]: This didn't work for me. Do we need a different URL?

File: README.md [hunk 2]
```diff
@@ -896,6 +901,81 @@ following articles:
...
+We can use a Docker compose file to simulate cluster mode. We'll start a feeder and two workers by default. To start the cluster, run the 
+sample docker-compose.yml file:
+```
+docker-compose -f contrib/docker/docker-compose.yml up
+```
+
+Once the cluster has started, we can manage input and output by looking at the volumes that were created by the compose file. The volume
+ location can be found in ```/var/lib/docker/volumes/```. Simply move a file into the input directory for Emissary to process it:
+```
+cp emissary-knight.png /var/lib/docker/volumes/docker_input/_data/InputData/
```
Comment (LightSeekerSC) [inline README.md:976]: I don't have a /var/lib/docker when doing this. Do I need to do something special or do I need to create this directory?

Comment (jpdahlke) [reply README.md:976]: It is permission controlled so you might need to change your perms

File: README.md (hunk 1 above)
Comment (jpdahlke) [reply README.md:964]: I'm pretty sure this worked for me

File: README.md (hunk 2 above)
Comment (LightSeekerSC) [reply README.md:976]: I am not quite sure I understand. Do I need to change the permissions of my /var/lib directory? Is this another step we should add to the README?

File: README.md (hunk 1 above)
Comment (LightSeekerSC) [reply README.md:964]: I still can't get to it. I am running this on an EC2 instance and then I am trying to access it through a firefox browser that is being X11 forwarded back from my EC2 instance. I get an "Unable to connect" error from firefox. Was there anything else additional you needed to do in order to enable this?

File: README.md (hunk 2 above)
Comment (dev-mlb) [reply README.md:980]: Are you running with sudo? Some of these commands do not work if not setup to run without sudo. This is specific to Docker setup, so it should be in the Docker docs. 

Updated the README to include that you need to run without sudo.

File: README.md (hunk 1 above)
Comment (dev-mlb) [reply README.md]: This sounds like an issue running a Docker container with X11 forwarding, not specific to this Docker image. I am running it locally on my box, so localhost:8001 is working fine for me. Unless there is a issue with the image itself, I'm marking this comment resolved.

File: README.md (hunk 2 above)
Comment (LightSeekerSC) [reply README.md:980]: After investigation, /var/lib/docker/volumes has root root 700 permissions. This prohibits me from executing the sudo command

I have setup my docker group and added myself (pretty sure I did this before running into this issue the first time). I looked into the post-installation instructions and I didn't see anything in direct regards to this path/issue.

What are your permissions for this directory?

Comment (dev-mlb) [reply README.md]: changed this to use docker cp

File: README.md [hunk 3]
```diff
@@ -908,76 +908,101 @@ with ```sudo```. Please see the linux-postinstall page in the Docker documentati
...
 docker-compose -f contrib/docker/docker-compose.yml up
 ```
 
-Once the cluster has started, we can manage input and output by looking at the volumes that were created by the compose file. The volume
- location can be found in ```/var/lib/docker/volumes/```. Simply move a file into the input directory for Emissary to process it:
+Use docker copy to run a file through Emissary:
+
 ```
-cp emissary-knight.png /var/lib/docker/volumes/docker_input/_data/InputData/
+docker cp emissary-knight.png docker_emissary-feeder_1:/opt/emissary/target/data/InputData/
```
Comment (LightSeekerSC) [inline README.md:1005]: This worked for me! Also, I was able to connect to the http endpoint. Not sure if I did something different or what happened there

This all checks out and I will approve, but I have two desired changes:
1) For http endpoint please add a line with what the username and password for the endpoint are or where to find that info
2) Explain where the emissary-knight.png goes (I couldn't seem to find the log or output data in the normal locations I would expect)

**Review by LightSeekerSC — APPROVED** [review]

**Pull request merged**